src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -514,7 +514,6 @@ public async Task<OrganizationSsoResponseModel> PostSso(Guid id, [FromBody] Orga
     }
 
     [HttpPut("{id}/collection-management")]
-    [SelfHosted(NotSelfHostedOnly = true)]
```

Comment:
This change isn't feature flagged - strictly speaking you'd have to check `!featureFlag && globalSettings.SelfHosted` in the body of the method and throw an error if required. However if we're just going to rely on the client-side flag for the ~1 release that this code is here but inactive, I think that's OK.

Reply:
I did decide in the moment that it was okay not to feature flag this, and in some ways was maybe even more risky since that involves modifying existing logic a bit, but I did go ahead and add this as suggested.

__Commit:__ [Feature toggle route unblocking](https://github.com/bitwarden/server/pull/4730/commits/2894dab5b3ed7828aaf03d6d09433fc5a15fa1f9)